Engine/Module/ModuleFileSystem.cpp

```diff
@@ -1,32 +1,59 @@
 #include "ModuleFileSystem.h"
+
+#include "Filesystem/PathAtlas.h"
 #include "Main/Application.h"
 #include "Module/ModuleResourceManager.h"
-#include <SDL/SDL.h>
+
 #include <algorithm>
-#include <cctype>
+#include <SDL/SDL.h>
+#include <stack>
+
+ModuleFileSystem::~ModuleFileSystem()
+{
+	CleanUp();
+}
+
+bool ModuleFileSystem::Init()
+{
+	APP_LOG_SECTION("************ Module File System Init ************");
+
+	//PHYSFS_permitSymbolicLinks(1);
```

Comment:
why is this commented?

Reply:
Because atm we are not accessing pathes in the `PhysFS` search path using symbolic links, I left it commented just in case we need it on the future